app/views/organization_timeline_entries/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if can?(:read, @organization_timeline_entry) %>
+  <%- model_class = OrganizationTimelineEntry -%>
+  <div class="page-header">
+    <h1><%=t '.title', :default => model_class.model_name.human %></h1>
```

Comment:
Copy the title from the index page for the organization so it says something like "Alpha Phi's Downtime". You will just need to pull the organization object in the controller.